src/views/Drawer.tsx

```diff
@@ -366,27 +369,31 @@ export default class DrawerView extends React.PureComponent<Props> {
         set(this.touchX, 0),
         this.transitionTo(
           cond(
-            or(
-              and(
-                greaterThan(abs(this.gestureX), SWIPE_DISTANCE_MINIMUM),
-                greaterThan(abs(this.velocityX), this.swipeVelocityThreshold)
-              ),
-              greaterThan(abs(this.gestureX), this.swipeDistanceThreshold)
-            ),
+            this.isSpringManuallyTriggered,
+            this.isOpen,
             cond(
-              eq(this.drawerPosition, DIRECTION_LEFT),
```

Comment:
This are just indentation changes right? I don't notice any differences

Reply:
yes, but this code is now more "nested" so I'd say it's expected